src/enums/filingCodes.ts

```diff
@@ -1,8 +1,9 @@
 export enum FilingCodes {
   ALTERATION = 'ALTER',
-  CHANGE_OF_REGISTRATION = 'FMCHANGE',
   CORRECTION = 'CRCTN',
-  CONVERSION = 'FMCONV',
+  FMCHANGE = 'FMCHANGE',
+  FMCONV = 'FMCONV',
+  FMCORR = 'FMCORR',
```

Comment:
I renamed these as they are specific to firms. This should also pre-empt future headaches when we implement more changes/conversions/corrections.